### ISSUE TITLE
RuntimeError: Pipe writer is misconfigured, named pipe path is not set

### ISSUE THREAD

**Issue by villeristi** [scrapinghub/scrapinghub-entrypoint-scrapy#52]
Trying to build a custom docker-image with:
```
1. scrapy startproject test
2. scrapy genspider testSpider
3. shub image init
4. docker build . -t customnamehere
```
Dockerfile:

```
FROM scrapinghub/scrapinghub-stack-scrapy:1.5-py3
ENV TERM xterm
ENV SCRAPY_SETTINGS_MODULE test.settings
RUN mkdir -p /app
WORKDIR /app
COPY ./requirements.txt /app/requirements.txt
RUN pip install --no-cache-dir -r requirements.txt
COPY . /app
RUN python setup.py install
```

command `docker run customnamehere shub-image-info` works like it should but `docker run customnamehere start-crawl` 

throws 

```RuntimeError: Pipe writer is misconfigured, named pipe path is not set```

Is there some more steps to properly configure or...?

**Comment by vshlapakov** [comment]
Hey @villeristi An ability to run custom images locally is not implemented/tested yet, and as of now, there's no a proper recommended way. But the feature is important indeed, and it's already in my priority list. I'll try to allocate some time during next weeks to add a separate command to shub tool to address your demands.

**Comment by villeristi** [comment]
Hi, any news about this?

**Comment by vshlapakov** [comment]
@villeristi yes, it was implemented in shub and released in `2.9.0`:
- https://github.com/scrapinghub/shub/releases/tag/v2.9.0
- https://github.com/scrapinghub/shub/pull/346
